docs/sw-update-conf.json

```diff
@@ -1,123 +1,121 @@
 {
-  "version": "1",
   "default": {
     "type": "github_commit",
     "user": "mrbeam",
     "stable": {
-      "branch": "mrbeam2-stable"
+      "branch": "mrbeam2-stable",
+      "branch_default": "mrbeam2-stable"
     },
     "beta": {
-      "branch": "mrbeam2-beta"
+      "branch": "mrbeam2-beta",
+      "branch_default": "mrbeam2-beta"
     },
     "develop": {
-      "branch": "develop"
+      "branch": "develop",
+      "branch_default": "develop"
+    },
+    "alpha": {
+      "branch": "mrbeam2-alpha",
+      "branch_default": "mrbeam2-alpha"
     },
     "restart": "environment"
   },
-  "octoprint": {
-    "pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-    "prerelease": false,
-    "develop":{
-      "prerelease": true
-    },
-    "beamos_date": {
-      "2021-06-11": {
+  "modules": {
+    "octoprint": {
+      "type": "github_release",
+      "develop": {
+        "type": "github_commit"
+      },
+      "beta": {
+        "prerelease_channel": "mrbeam2-beta",
+        "prerelease": true
+      },
+      "alpha": {
+        "prerelease_channel": "mrbeam2-alpha",
         "prerelease": true
       }
-    }
-  },
-  "mrbeam": {
-    "name": " MrBeam Plugin",
-    "repo": "MrBeamPlugin",
-    "restart": "octoprint",
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeam": {
+      "name": " MrBeam Plugin",
+      "repo": "MrBeamPlugin",
+      "restart": "octoprint",
+      "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
+      "dependencies": {
+        "mrbeam-ledstrips": {
+          "name": "MrBeam LED Strips",
+          "repo": "MrBeamLedStrips",
+          "pip": "https://github.com/mrbeam/MrBeamLedStrips/archive/{target_version}.zip",
+          "global_pip_command": true,
+          "beamos_date": {
+            "2021-06-11": {
+              "pip_command": "sudo /usr/local/mrbeam_ledstrips/venv/bin/pip"
+            }
+          }
+        },
+        "iobeam": {
+          "name": "iobeam",
+          "type": "bitbucket_commit",
+          "repo": "iobeam",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/iobeam.git@{target_version}",
+          "global_pip_command": true,
+          "api_user": "MrBeamDev",
+          "api_password": "v2T5pFkmdgDqbFBJAqrt",
```

Comment:
Is it fine to have the password here? shouldn't it be somewhere more safe?

Reply:
but where? before it was also open in the MrBeamPlugin repo

Reply:
I guess it would be anyway a different ticket.. We could bring it up later on...